src/main/java/com/example/buysell/domains/Image.java

```diff
@@ -0,0 +1,106 @@
+package com.example.buysell.domains;
+
+
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+
+
+
+
+@Entity
+public class Image {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
```

Comment:
```suggestion
    @Column(name="image_name", length=255, nullable=false, unique=false)
    private String name;
```
Также нужно явно указывать ограничения для колонки.
Если у тебя не может быть записи в базе с нулевым именем (image_name = NULL), т.е. имя должно быть обязательным, то тут в аннотации обязательно нужно это прописывать.

Reply:
Ну и имена колонок в БД не должны быть camelCase. Обычно это ловеркейс с нижним подчеркиванием в качестве разделителя. Возможно JPA так и делает, если видит имя originalFileName → original_file_name.